src/lib.rs

```diff
@@ -143,7 +143,7 @@ pub fn broadcast<T>(cap: usize) -> (Sender<T>, Receiver<T>) {
 
```

Comment:
A bit of summary about the changes in the commit log (`Add CapacityLimiter trait`) please: The types added and brief description e.g